migrations/sqls/20201120135927-create-charge-versions-metadata-table-up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TYPE charge_version_metadata_status AS ENUM('current', 'superseded');
+
+CREATE TABLE water_import.charge_versions_metadata
+(
+	external_id varchar not null,
+	start_date timestamp not null,
+	end_date timestamp,
+	status charge_version_metadata_status not null,
+	is_nald_gap boolean not null
```

Comment:
Wondering if it would be handy to have `date_created` and `date_updated` timestamp columns - this will make it easier to know if the importer is running